packages/passport/sdk/src/zkEvm/zkEvmProvider.ts

```diff
@@ -464,6 +465,42 @@ export class ZkEvmProvider implements Provider {
       case 'eth_getTransactionReceipt': {
         return this.#rpcProvider.send(request.method, request.params || []);
       }
+      case 'im_signEjectionTransaction': {
+        const zkEvmAddress = await this.#getZkEvmAddress();
+        if (!zkEvmAddress) {
+          throw new JsonRpcError(
+            ProviderErrorCode.UNAUTHORIZED,
+            'Unauthorised - call eth_requestAccounts first',
+          );
+        }
+
+        const flow = trackFlow('passport', 'imSignEjectionTransaction');
+
+        try {
+          return await this.#guardianClient.withConfirmationScreen({
+            width: 480,
+            height: 720,
+          })(async () => {
```

Comment:
This flow shouldn't use the confirmation screen, so we can scrap this wrapping `withConfirmationScreen` call